lib/portlayer/storage/vsphere/volume.go

```diff
@@ -189,8 +203,26 @@ func (v *VolumeStore) VolumeCreate(ctx context.Context, ID string, store *url.UR
 	return vol, nil
 }
 
-func (v *VolumeStore) VolumeDestroy(ctx context.Context, ID string) error {
-	return fmt.Errorf("TBD.  Not supported yet")
+func (v *VolumeStore) VolumeDestroy(ctx context.Context, vol *storage.Volume) error {
+	if err := volumesInUse(ctx, vol.ID); err != nil {
```

Comment:
Ok it makes sense to keep the call to `volumesInUse` for the first case. The other two cases are really the same.